config/rbac/role.yaml

```diff
@@ -181,6 +181,14 @@ rules:
   - get
   - list
   - watch
+- apiGroups:
+  - apiextensions.k8s.io
+  resources:
+  - customresourcedefinitions
```

Comment:
This rbac will get applied to the management cluster when we install the operator, but we don't really need these permissions there. Or at least we don't need `create` there, right?

Reply:
Right, basically we need list/get for the in-cluster and create from the external cluster. I'll try to separate these.